lib/matplotlib/collections.py

```diff
@@ -1262,62 +1262,60 @@ def __init__(self,
                  **kwargs
                  ):
         """
-        *positions*
-            a sequence of numerical values or a 1D numpy array.  Can be None
+        Parameters
+        ----------
+        positions : 1D array-like object or None.
```

Comment:
Yes you're right, I will revert my changes toward your idea of just fixing the docstring.